nss-day-cohort-60/rare-client-tenacious-tators
# Lh filter by user

Closes #12


# Description of Proposed Changes

Added a filter by Author drop down
Moved both dropdowns to individual modules
created PostContainer module where drop downs and rendered list of posts are rendered.
Updated ApplicationViews to reflect new module structure
---


# Steps to Test

1. Fetch the [lh-filter-by-user](https://github.com/nss-day-cohort-60/rare-client-tenacious-tators/compare/main...lh-filter-by-user?expand=1) branch and switch to it
2.login and navigate to /posts
3. use Select By Author dropdown to select author and verify you are getting the correctly filtered results
4. use Select By Author (choose an author with posts with more than one category) and Select by Category and verify you get results filtered by both criteria
5. Choose View All in both drop downs and verify you see all posts
